css/mtt.css

```diff
@@ -0,0 +1,102 @@
+@font-face {
+	  font-family: 'Multicolore';
+	  src: url(/fonts/multicolore/Multicolore.otf);
+  }
+
+@font-face {
+  font-family: 'Biko';
+  src: url('/fonts/biko/Biko_Regular.otf');
+}
```

Comment:
Are this and the previous rule defined anywhere else too?  I just wonder if this is a duplicate rule and it would be better to be defined in the global CSS.

Also, it might make sense to be consistent about if `url` will be quoted or not.  And the indentation of CSS rules (plus if there is a space before `{` or not)

Reply:
I think it makes sense to make this a global css rule. Where are global css rules defined?

Reply:
It seems that they are modified in the `_scss` directory which gets compiled into the `bootstrap.css` file in the `css` directory. https://github.com/EnvironmentalDashboard/environmentaldashboard.org/blob/master/scss/bootstrap.scss may be the file you are after - some fonts are defined at the top.

I figure compiling Bootstrap from SCSS is a common process that you can then look up online.